odoo/addons/base/controllers/rpc.py

```diff
@@ -68,6 +68,7 @@ def _xmlrpc(self, service):
         data = request.httprequest.get_data()
         params, method = loads(data)
         result = dispatch_rpc(service, method, params)
+        _logger.debug('XMLRPC call %s with params : %s.', method, params)
```

Comment:
It sounds nice, can you create the variable _logger on the top of the file to merge this ?
Thx @BHanq